spec/models/project_spec.rb

```diff
@@ -36,6 +36,51 @@
     end
   end
 
+  describe "GitHub API interaction" do
+    let(:project) { build(:project) }
+    let(:args) { project.github_api_args.merge!(
```

Comment:
Avoid using {...} for multi-line blocks.<br>Block body expression is on the same line as the block start.